src/main/java/com/hubspot/jinjava/lib/tag/AutoEscapeTag.java

```diff
@@ -21,16 +21,20 @@
                 "{% endautoescape %}")
     })
 public class AutoEscapeTag implements Tag {
+
+  public static final String TAG_NAME = "autoescape";
+  public static final String END_TAG_NAME = "endautoescape";
```

Comment:
`END_TAG_NAME` can always be "end" + `TAG_NAME`;

Reply:
I was trying to come up with the best way to do this. Should we just default to the endtag being "end" + TAG_NAME and hope that people override the method for the null case?

Reply:
Ok now we have `default String getEndTagName()` on `Tag.class` since I only ready need to use the start tag for building exclusion lists.